DEVELOPMENT.md

```diff
@@ -176,7 +176,7 @@ kubectl apply -f ./third_party/istio-1.3-latest/istio-knative-extras.yaml
 1. Deploy `cert-manager` CRDs
 
    ```shell
-   kubectl apply -f ./third_party/cert-manager-0.9.1/cert-manager-crds.yaml
+   kubectl apply -f ./third_party/cert-manager-0.12.0/cert-manager-crds.yaml
```

Comment:
We should update the version in `ko delete` below in this docs https://github.com/knative/serving/pull/6609/files#diff-f785e9e46b9afd2567df7a658338db17R295-R296 ?

Reply:
Done